ci/tasks/generate-account-management-deployment-variables.yml

```diff
@@ -14,6 +14,7 @@ outputs:
 params:
   SESSION_EXPIRY: ((build-session-expiry))
   SESSION_SECRET: ((build-session-secret))
+  GTM_ID: ((build-gtm-id))
```

Comment:
Does it cause any problems if this is unset?  Will be unset for local.

Reply:
it will just be undefined